CorsixTH/Src/th_gfx_sdl.cpp

```diff
@@ -37,6 +37,8 @@ SOFTWARE.
 
 #include "th_map.h"
 
+#define PI 3.14159265
```

Comment:
This may be undesired and instead use actual pi (but who am I to know).

Reply:
Nope, not until c++ 20 and were at 14. Should be constexpr pi = ... though.

Reply:
Already ahead of you :)